autodoc/admin-api/data/admin-api.lua

```diff
@@ -669,6 +669,37 @@ return {
               means Kong does not currently have a valid configuration loaded.
           ]],
         },
+      },
+      ["/status/ready"] = {
```

Comment:
This doesn't seem right. We need to put this endpoint on the status API, not Admin API.

Reply:
@hbagdi Sorry but seems status APIs are a part of Admin API, and it is in the health routes section of Admin API documentation. If my understanding is incorrect, please correct me.